src/lib/components/02-components/callout/callout.tsx

```diff
@@ -109,8 +111,16 @@ export default function Callout({
   ) : null;
 
   return (
-    <div id={containerId} className={cn(colorSchema[color].wrapper, className)} {...props}>
-      <div className="mx-auto flex max-w-screen-lg flex-col-reverse md:flex-row md:items-center md:gap-lg xl:gap-xl">
+    <div
+      id={containerId}
+      className={cn(!displayAsBanner && colorSchema[color].wrapper, className)}
+      {...props}>
+      <div
+        className={cn(
+          'mx-auto flex max-w-screen-lg flex-col-reverse md:flex-row md:items-center md:gap-lg xl:gap-xl',
+          displayAsBanner && colorSchema[color].wrapper,
+          displayAsBanner && 'overflow-hidden rounded-[40px] shadow-sm',
```

Comment:
Can't you add `overflow-hidden rounded-[40px] shadow-sm` to `colorSchema[color].wrapper`?